cfgov/jinja2/v1/rural-or-underserved/index.html

```diff
@@ -294,20 +294,24 @@ <h2 class="js-table-title">
                             First-lien covered transactions extended in <span class="chosenYear">2015</span> that are secured by these properties can be counted as rural or underserved in your determination of your first-lien covered transactions in <span class="chosenYear">2015</span> that are secured by properties located in rural or underserved areas.
                         </p>
 
-                        <table class="table" style="border-width: 1px;" width="100%">
-                            <thead>
-                                <tr>
-                                <th width="31.5%">Address Entered</th>
-                                <th width="31.5%">Address Identified</th>
-                                <th width="15%">County</th>
-                                <th width="22%">Rural or Underserved?</th>
-                                </tr>
-                            </thead>
-                            <tbody class="data"></tbody>
-                        </table>
-
-                        <a data-table="rural" class="button-more u-hidden a-btn a-btn__secondary no-decoration" href="#" id="ruralMore">View 10 more</a>
-                        <a data-table="rural" class="margin-left-1 view-all u-hidden no-visit" href="#" id="ruralAll">View all</a>
+                        <div class="o-table o-table-wrapper__scrolling">
```

Comment:
It looks like our scrolling tables have zebra striping baked in. Ordinarily adding zebra striping seems fine to me, but in this case I think we might want to remove it since it gets weird in the top two result tables. Every other row in those tables is initially hidden since they contain the expandable map, so when the results load it looks like those tables aren't striped at all. Then when you expand the map for a row, the results row is not striped but it's corresponding map row is. So maybe overriding the default zebra striping is the way to go here? Alternatively, I'd also be fine if we got zebra striping working in the top two results tables in a way that (a) showed when the results load and (b) striped a result row and its corresponding map row the same color.

Reply:
I overrode the zebra striping and added a TODO for adjusting the styles in cf-tables.

Reply:
Looks good 👍